laws/shared/src/main/scala/algebra/laws/OrderLaws.scala

```diff
@@ -112,6 +114,75 @@ trait OrderLaws[A] extends Laws {
     }
   )
 
+  def signed(implicit A: Signed[A]) = new OrderProperties(
+    name = "signed",
+    parent = Some(order),
+    "abs non-negative" -> forAll((x: A) => A.sign(A.abs(x)) != Signed.Negative),
```

Comment:
Do we require that abs(x) >= x?

Seems natural to me, but I don't see that is implied.

Reply:
Sure!